### ISSUE TITLE
Reversed order of changes when inserting many at once

### ISSUE THREAD

**Issue by mariuszjamro** [christiandelbianco/monitor-table-change-with-sqltabledependency#124]
I noticed that when inserting multiple rows at once events dispatched by `OnChanged` are returned in reversed order. I think the order should match order rows in the INSERT query.

**How to reproduce:**

1. Create table
```
CREATE TABLE [dbo].[Foo](
	[Id] [int] IDENTITY(1,1) NOT NULL,
	[Name] [nchar](10) NULL
) ON [PRIMARY]
```
Note: Id column is auto incremented. Each new row should have [Id] greater then the previous one

2. Run the program

```

public class Program
{
    public class Foo
    {
        public long Id { get; set; }
        public string Name { get; set; }
    }

    private static string _con = "Server=.\\SQLEXPRESS;Database=FooDb;Trusted_Connection=True;MultipleActiveResultSets=true";

    public static void Main()
    {
        using (var dep = new SqlTableDependency<Foo>(_con, "Foo"))
        {
            dep.OnChanged += Changed;
            dep.Start();

            Console.WriteLine("Press a key to exit");
            Console.ReadKey();

            dep.Stop();
        }
    }

    public static void Changed(object sender, RecordChangedEventArgs<Foo> e)
    {
        var changedEntity = e.Entity;

        Console.WriteLine("DML operation: " + e.ChangeType);
        Console.WriteLine("ID: " + changedEntity.Id);
        Console.WriteLine("Name: " + changedEntity.Name);
    }
}
```

3. Trigger insert of multiple rows:

```
INSERT INTO [dbo].[Foo] ([Name]) VALUES ('a'),('b'),('c')`
```

Current outcome - ID's are in descending order:

```
DML operation: Insert
ID: 3
Name: c   <---- 'c' is first
DML operation: Insert
ID: 2
Name: b
DML operation: Insert
ID: 1
Name: a
```

Expected outcome - ID's are in ascending order:

```
DML operation: Insert
ID: 1
Name: a  <---- 'a' is first
DML operation: Insert
ID: 2
Name: b
DML operation: Insert
ID: 3
Name: c
```

**Comment by christiandelbianco** [comment]
Hi,

I will check. Someone else already reported me this behaviour. And i remember that there was a specific technical reason for that.

**Comment by christiandelbianco** [comment]
Hi,

this issue has been fixed.
Check test class **MultiDmlOperationsOrderTest.cs**.

It will released with version **8.5.4**

Regards

**Comment by christiandelbianco** [comment]
Hi, as a different error was found (https://github.com/christiandelbianco/monitor-table-change-with-sqltabledependency/issues/146), the version 8.5.4 has been released. It contains also your Fix.
Regards
